translate/translate.js

```diff
@@ -81,12 +76,37 @@ function listLanguages (apiKey, callback) {
 }
 // [END list_languages]
 
+// [START list_languages_target]
```

Comment:
Hmm - I feel like `listLanguages()` and `listLanguagesTarget()` could be combined into one function if `listLanguagesTarget()` was given a default `target` of `English`.

Thoughts?